client/src/components/Main/NoteArea.tsx

```diff
@@ -1,14 +1,47 @@
+import { NotesProps } from "../../App";
 import AddTag from "./AddTag";
-import NoteEditor from "./NoteEditor";
-import NoteTitle from "./NoteTitle";
 
-const NoteArea = () => {
+type NoteAreaProps = {
+  activeNote: NotesProps | null;
+  updateNote: (e: any, id: number) => void;
+};
+
+const NoteArea = ({ activeNote, updateNote }: NoteAreaProps) => {
+  if (!activeNote) {
+    return (
+      <p className="flex-1 flex justify-center h-screen items-center border border-t-0 rounded-br">
+        No note selected.
+      </p>
+    );
+  }
+
+  if (!activeNote?.title) activeNote.title = "";
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/zerexei/zoop-note/pullRequest?prid=7533483)
```suggestion
  if (!activeNote?.title) {activeNote.title = "";}
```